lib/modules/vm_connection.rb

```diff
@@ -16,7 +16,7 @@ class VMConnection
     ";"
 
   SYSTEM_DEPENDENCIES =
-    'export PATH="$PATH:$HOME/.rvm/bin"; source /home/miguel/.rvm/scripts/rvm'.freeze
+    'export PATH="$PATH:$HOME/.rvm/bin"; source /Users/Markus.Springer/.rvm/scripts/rvm'.freeze
```

Comment:
Yes that might just work. Otherwise, it would not even be a secret but just a config